Extensions/3D/CustomRuntimeObject3D.ts

```diff
@@ -34,7 +34,6 @@ namespace gdjs {
       objectData: gdjs.Object3DData & gdjs.CustomObjectConfiguration
     ) {
       super(parent, objectData);
-      this._renderer.reinitialize(this, parent);
```

Comment:
This call was probably to add a default layer, but it's already done in `CustomRuntimeObject` constructor by calling `_initializeFromObjectData` --> `loadFrom`.
This call was bug prone because `reinitialize` is supposed to clear the content which is already loaded at this point.

Reply:
Yeah it was weird to call reinitialize here. Better without it.